app/code/core/Mage/CatalogRule/Model/Action/Index/Refresh.php

```diff
@@ -118,7 +118,7 @@ public function execute()
 
         /** @var Mage_Core_Model_Date $coreDate */
         $coreDate  = $this->_factory->getModel('core/date');
-        $timestamp = $coreDate->gmtTimestamp('Today');
+        $timestamp = Mage::app()->getLocale()->date(null, null, null, true)->get(Zend_Date::TIMESTAMP);
```

Comment:
Code wise, `$timestamp = Mage::app()->getLocale()-> storeTimeStamp();` is may be clearer.  Also lines 119 and 120 should be removed.